consensus/service/src/config/tokens.rs

```diff
@@ -0,0 +1,524 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Tokens configuration.
+
+use crate::consensus_service::ConsensusServiceError;
+use mc_common::HashSet;
+use mc_consensus_enclave::FeeMap;
+use mc_transaction_core::{tokens::Mob, Token, TokenId};
+use serde::{Deserialize, Serialize};
+use std::{fs, iter::FromIterator, path::Path};
+
+/// Single token configuration.
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
+pub struct TokenConfig {
+    /// Token ID.
+    token_id: TokenId,
+
+    /// Minimum fee, in the smallest denomination supported by the token (e.g.
+    /// picomob for the MOB token).
+    minimum_fee: Option<u64>,
+
+    /// Allow extreme fees. Currently the limitation is only enforced for MOB
+    /// (>= 1MOB, <= 0.000_000_01 MOB).
+    // instructs serde to default to false without explicitly requiring this field to appear in the
+    // file.
+    #[serde(default)]
+    allow_any_fee: bool,
```

Comment:
Would it make sense to replace this with an optional `maximum_fee`? It seems like the low end of the range is already defined by minimum_fee.